FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater

```diff
@@ -0,0 +1 @@
+/research/ece/lnis/CAD_TOOLS/DKITS/skywater
```

Comment:
Symbolic link should not be uploaded.

Reply:
You can remove this, or may be add in .gitignore

Reply:
I can remove but I will have to modify (pollute) your branch. It is up to you.

Reply:
I will push update on this branch in sometime